### PR TITLE
Add back button

### PR DESCRIPTION
**Pull request by samshara**

### CODE REVIEW

**Review by tnagorra — CHANGES_REQUESTED** [review]
image

Tooltip "Show table" doesn't make sense now.
Should have been "Go to leads" or something similar.

File: src/views/LeadsViz/index.js [hunk 1]
```diff
@@ -348,18 +345,20 @@ export default class LeadsViz extends React.PureComponent {
...
-                <footer className={styles.footer}>
                     <Link
-                        className={styles.link}
+                        className={styles.backLink}
                         to={reverseRoute(pathNames.leads, { projectId: activeProject.id })}
-                        replace
+                        title={_ts('leadsViz', 'showTable')}
                     >
-                        { _ts('leadsViz', 'showTable') }
+<<<<<<< Updated upstream
```
Comment (AdityaKhatri) [inline src/views/LeadsViz/index.js]: LoL

File: src/views/LeadsViz/index.js [hunk 2]
```diff
@@ -348,18 +345,20 @@ export default class LeadsViz extends React.PureComponent {
...
+                        <i className={iconNames.back} />
+=======
+                        { _ts('leadsViz', 'showLeads') }
+>>>>>>> Stashed changes
```
Comment (AdityaKhatri) [inline src/views/LeadsViz/index.js]: LOLOL

File: src/views/LeadsViz/index.js [hunk 3]
```diff
@@ -348,18 +345,20 @@ export default class LeadsViz extends React.PureComponent {
...
-                        className={styles.link}
+                        className={styles.backLink}
                         to={reverseRoute(pathNames.leads, { projectId: activeProject.id })}
-                        replace
+                        title={_ts('leadsViz', 'showTable')}
                     >
-                        { _ts('leadsViz', 'showTable') }
+<<<<<<< Updated upstream
+                        <i className={iconNames.back} />
+=======
```
Comment (AdityaKhatri) [inline src/views/LeadsViz/index.js]: LOL

**Pull request closed without merging**